UI/data/locale/en-US.ini

```diff
@@ -86,6 +86,8 @@ HideTransition="Hide Transition"
 None="None"
 StudioMode.Preview="Preview"
 StudioMode.Program="Program"
+StudioMode.PreviewSceneName="Preview - %1"
```

Comment:
Using a colon instead of a dash "Preview: %1" and "Program: %1" would make it clearer that the scene name is information about the Preview or Program. It also saves a space, if you are worried about long names

Reply:
I think the dash look better. We're not worried about long names, but about ones spanning multiple lines.

Reply:
Just my suggestion. I'll be happy to see the titles regardless of punctuation.
The colon format is used by "Star Wars: the Last Jedi" and over a dozen items in en-US.ini, such as
- QuickTransitions.HotkeyName="Quick Transition: %1"
- Basic.Settings.Stream.Recommended.MaxVideoBitrate="Maximum Video Bitrate: %1 kbps"
- Basic.Settings.Advanced.StreamDelay.MemoryUsage="Estimated Memory Usage: %1 MB"
- YouTube.AuthError.Text="Failed to get channel information: %1."